### ISSUE TITLE
IDE X - Word-Right behavior

### ISSUE THREAD

**Issue by cdcmicro** [Komodo/KomodoEdit#1791]
### Short Summary

The default behavior of Ctrl-Right appears to be  "Editor: Go to End of word".  The Keybinding preference UI doesn't show it.

### Steps to Reproduce

Editing text, press Ctrl-Right.

### Expected results

Cursor moves to the beginning of the next word.

Keybinding UI shows "Editor: Go to End of word" bound to Ctrl-Right

### Actual results

Cusror moves to the next end of word position.

Keybinding UI shows "Editor: Go to End of word" bound to Ctrl-E.

When I attempt to add Ctrl-Right to "Editor: Move One Word Right", it indicates Ctrl-Right is already assigned to Editor: Go to End of word.

### Platform Information

Komodo IDE, version 10.0.1, build 89237, platform win32-x86.
Built on Wed Jun 15 10:57:56 2016.

### Additional Information

It's an easy workaround to change the Ctrl-Right keybinding.  I think Ctrl-Right should be bound to "Editor: Move One Word Right" by default, to match Ctrl-Left, and to match the behavior of IDE9 and earlier (on Windows, anyway).

See #1792

**Comment by mitchell-as** [comment]
Hi, thanks for the report.

On Linux, Ctrl+Right in any text field or other text editing application matches Komodo's Ctrl+Right ("Editor: Go to End of Word"), so at least for Linux this should not be changed.

@cgchoffman you're better apt to judge the default behavior on Windows for Ctrl+Right in text entries and other applications and see if Komodo's keybinding on Windows should be changed.

I do note that Ctrl+Right and Ctrl+E are bound to the same "Editor: Go to End of Word" command, and that is probably not intended, so perhaps that needs to be fixed anyway.

**Comment by cdcmicro** [comment]
Ctrl+Right => Word-right is a Windows convention.  Left, too.

**Comment by cgchoffman** [comment]
I agree with you @cdcmicro.  And thanks for the heads up about Linux @mitchell-as.  @cdcmicro You can use the legacy key binding set for now.  It is the binding set from Komodo < X.

**Comment by cdcmicro** [comment]
Strongly related, IMO, is Ctrl+Shift+Right.  In IDE 10.1.1, it selects to the end of the next word.  The Windows convention is to select to the beginning of the next word.

**Comment by cdcmicro** [comment]
Clicked the wrong button :-(

**Comment by Naatan** [comment]
> Ctrl+Right => Word-right is a Windows convention. Left, too.

Linux too. And on OSX it's Alt+Right. So definitely not something unique to windows.

**Comment by cdcmicro** [comment]
Remapped it in my setup.  Was

>Editor: Select to End of Word

Now

>Editor: Select Next Word
